baselines/table-transformer/README.md

```diff
@@ -0,0 +1,35 @@
+# table-transformer baseline
```

Comment:
Can you please also update the parent README by removing the `[coming soon]` note by this baseline?